engine/admin/ansible.md

```diff
@@ -0,0 +1,72 @@
+---
+aliases:
```

Comment:
Since we're now using Jekyll, the `aliases` and `menu` properties are no longer needed. You can delete them.